frontend/src/BudgetCard.tsx

```diff
@@ -0,0 +1,225 @@
+                sx={{ flexGrow: 1 }}
+            >
+                {
+                    <>
+                        <BudgetRow hiddenColumns={hiddenColumns}>
+                            <div />
+                            <HeaderLabel text="Budget" />
+                            <HeaderLabel text="Reserved" />
+                            <HeaderLabel text="Remaining" />
+                        </BudgetRow>
+
+                        {budgetFilters.has(BudgetFilter.RESOURCES) && (
+                            <BudgetRow hiddenColumns={hiddenColumns}>
+                                <HeaderLabel text="Resources" />
+                                <BudgetInput
+                                    label="Resource Budget"
+                                    isLabelVisible={false}
+                                    value={resourceBudget}
+                                    onChange={setResourceBudget}
+                                />
+                                <div />
+                                <OverUnderNumber
+                                    value={
+                                        resourceBudget -
+                                        getTotalCost(unitCounts)
+                                    }
+                                />
+                            </BudgetRow>
+                        )}
+
+                        {budgetFilters.has(BudgetFilter.PRODUCTION_LIMIT) && (
+                            <BudgetRow hiddenColumns={hiddenColumns}>
+                                <HeaderLabel text="Production Limit" />
+                                <BudgetInput
+                                    label="Production Limit Budget"
+                                    isLabelVisible={false}
+                                    value={capacityBudget}
+                                    onChange={setCapacityBudget}
+                                />
+                                <div />
+                                <OverUnderNumber
+                                    value={
+                                        capacityBudget -
+                                        sumUnitCounts(unitCounts)
+                                    }
+                                />
+                            </BudgetRow>
+                        )}
+
+                        {budgetFilters.has(BudgetFilter.FLEET_SUPPLY) && (
+                            <BudgetRow hiddenColumns={hiddenColumns}>
+                                <HeaderLabel text="Fleet Supply" />
+                                <BudgetInput
+                                    label="Fleet Supply Budget"
+                                    isLabelVisible={false}
+                                    value={maxFleetSupply}
+                                    onChange={setMaxFleetSupply}
+                                />
+                                <BudgetInput
+                                    label="Reserved Fleet Supply"
```

Comment:
I used the label "Reserved" to represent "already in use" fleet supply and ship capacity, but I was also trying out "Pre-allocated" or "Earmarked" or just "In Use" - couldn't decide what was the right balance of wordiness and easiness-to-understand. Super open to changing it!

Reply:
Yes I love that! I changed to "In Use" (which in hindsight is both the fewest number of characters and easiest to understand) and created a little `InfoIcon` component to display next to the column header. On a computer the tooltip will show on hover, and on a phone will show on click.

![image](https://github.com/user-attachments/assets/5c9c9adb-35d6-4e04-8404-38a5d319431c)